src/components/connections.vue

```diff
@@ -8,6 +8,7 @@ export default {
   name: 'connections',
   data() {
     return {
+      display: false,
```

Comment:
Would you consider a more specific variable name, like `displayNewConnectionForm`? Ideally this connections component should be split in two and the new connection form should be on its own component — then the variable name `display` would make a lot of sense. But here it's hard to distinguish what `display` refers to.